<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlogEntriesWidget</class>
 <widget class="QWidget" name="BlogEntriesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="BlogEntriesCalendarSplitter_">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="LeechCraft::Blogique::CalendarWidget" name="BlogEntriesCalendar_">
      <property name="firstDayOfWeek">
       <enum>Qt::Monday</enum>
      </property>
      <property name="gridVisible">
       <bool>true</bool>
      </property>
      <property name="dateEditEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Filter:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="BlogEntriesFilter_"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="BlogEntriesView_">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="CalendarVisibility_">
     <property name="text">
      <string>Show calendar</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
     <item>
      <widget class="QToolButton" name="LoadBlogEntries_">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Show last entries.</string>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RemoveBlogEntry_">
       <property name="toolTip">
        <string>Post will be removed from local storage and from user's blog.</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LeechCraft::Blogique::CalendarWidget</class>
   <extends>QCalendarWidget</extends>
   <header>calendarwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>CalendarVisibility_</sender>
   <signal>toggled(bool)</signal>
   <receiver>BlogEntriesCalendar_</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
